incubator/java-microprofile/image/Dockerfile-stack

```diff
@@ -30,12 +30,10 @@ ENV APPSODY_WATCH_REGEX="^.*(.xml|.java|.properties)$"
 
```

Comment:
above.. is mounting the m2 repo, you should really also be adding
ENV APPSODY_USER_RUN_AS_LOCAL=true
otherwise any manipulation of the m2 repo during use of the stack during run/debug/test will leave root owned files in the users m2 repo, which is a BAD idea.

Reply:
I added this. The only issue is now the pom.xml and src are owned by the UID and the target directory is owned by root. Prevents the classes directory from being created.. trying to work around this..